README.md

```diff
@@ -34,6 +34,12 @@ Invoke `gulp` to rebuild your app and regenerate the script that offlines it. In
 gulp && gulp deploy
 ```
 
+At least one commit to the repository is required for successful deploy.  The following could be used to commit the changes by Oghliner to the respository:
```

Comment:
Nit: respository -> repository

Otherwise looks great!

@marco-c You also reviewed this, can you merge it if you're happy with it?